docs/sidebars.js

```diff
@@ -295,12 +287,11 @@ module.exports = {
         "concepts/structure/overview",
         "concepts/structure/instance",
         "concepts/structure/organizations",
-        "concepts/structure/policies",
         "concepts/structure/projects",
         "concepts/structure/applications",
         "concepts/structure/granted_projects",
         "concepts/structure/users",
-        "concepts/structure/managers",
```

Comment:
@peintnermax Why did you remove managers?

Reply:
thought that the guide would be enough.. should I add a small description and then link to guides?

Reply:
which guide?

Reply:
I don't get what this has to do with updating the console guide?

Reply:
readded it

Reply:
not sure how fast to review 111 files when there's stuff like this inside. will take me some time.

Reply:
apart of this, which has now been readded, no link has been removed. The content fo the concepts has been reduced but now contains a link to console guides for more info so it should be alright..

Reply:
@mffap we moved everything that described the console to guides/console, in the concepts we do only have the description about what each "object" means.

I think you don't need to go through all the changes. maybe just have a quick look into the structure. 

@peintnermax I think we need to look at everything that was in the guides has still a page and a redirect to the console guide. The managers we might can have in both parts.